.github/workflows/_build-windows.yml

```diff
@@ -34,10 +34,14 @@ jobs:
           cmake --build . --config "Release" -- /maxcpucount /property:BuildInParallel=true /property:CL_MPCount=2 /verbosity:minimal
       - name: Test
         shell: pwsh
+        working-directory: build
         run: |
-          pushd build
           $env:Path += ";$($pwd.Path)\\Release"
           cmake --build . --config "Release" --target unittest
+      - name: Test
+        shell: pwsh
+        working-directory: build
+        run: |
           $env:FO_CHECKSUM_SPECIES_NAME = "SP_CELESTEPHYTE"
```

Comment:
Should we create a custom file species that will not be a part of the game and modify it?  
For the sake of the parser testing, we could add all possible effects and conditions there.  (not in this review)

Reply:
As I remember I tried to test crash on parsing and outputting dump.